docs/conf/sponsorship/prospectus.rst

```diff
@@ -0,0 +1,214 @@
+Write the Docs 2017
+####################
+
+Sponsorship
+###########
+
+.. raw:: pdf
+
+   PageBreak
+
+Concept
```

Comment:
I have a feeling this section is a bit wordy and duplicates a lot of our generic info, but unfortunately I can't think of a good way to trim it down... ideas welcome!